app/pages/home-logged-in.cjsx

```diff
@@ -60,6 +60,56 @@ FeaturedProjects = React.createClass
       <Link to="/projects" className="call-to-action standard-button x-large"><Translate content="home.featuredProjects.button" /></Link>
     </div>
 
+RecentProjects = React.createClass
+  displayName: 'RecentProjects'
+  
+  getInitialState: ->
+    projects: []
+    activityCounts: {}
+    title: ""
+    href: ''
+    action: ""
+  
+  componentWillMount: ->
+    @props.user
+      .get("project_preferences", page_size: 10, sort: '-updated_at')
```

Comment:
seems this end point should be able to filter on activity counts instead of filtering later...thoughts?